skrypty/people/color_people.lua

```diff
@@ -177,3 +206,39 @@ function scripts.people:trigger_people_starter()
     end
 end
 
+
+function scripts.people:process_line(msg)
+    if table.is_empty(self.tokens_table) then
+        return
+    end
+    local tokens = ansi2string(msg):gsub("%.", ""):gsub("[,!?-]", ""):gsub("\t", ""):gsub("\n", ""):split("[ /]")
```

Comment:
ansi2string(msg):lower() and remove all other :lower()

Reply:
Nie da rady, potrzebne są później wielkości znaków.